microsite/blog/2022-05-13-techdocs-addon-framework.md

```diff
@@ -0,0 +1,48 @@
+---
+# prettier-ignore
+title: "Introducing the TechDocs Addon Framework"
+author: Gary Niemen and Eric Peterson, Spotify
+authorURL: https://github.com/garyniemen/
+authorImageURL: https://github.com/garyniemen/.png
+authorURL: https://github.com/iamEAP/
+authorImageURL: https://github.com/iamEAP/.png
```

Comment:
The author images seems broken.. maybe update to `https://avatars.githubusercontent.com/u/65337273?v=4` and `https://avatars.githubusercontent.com/u/3496491?v=4`

I'm also not sure if multiple authors is supported for the version of docusaurus we use.. at least it wasn't last time me and Kat wrote one together.

Reply:
we added a note at the top of the blog in case you want to do that too, you can see here 

http://localhost:3000/blog/2022/01/25/backstage-homepage-templates

Reply:
If you are up for it, feel free to adjust the authorship - you can make me the author. And then add similar to the one you pointed to adding Eric. And then adjust the DCO so that it will work. I have noted the new expression of it for next time. If you haven't got time, let me know how to do it - is it a new commit to the same branch or can I do it straight in the PR or something?

Reply:
Also we don't want to do the merge until next week when Backstage 1.2 is released.

Reply:
all fixed, also added your note around 1.2 release to the PR description.